common/selectors/colorSelectors.js

```diff
@@ -69,8 +56,8 @@ const selectSiteIcon = createSelector(
   site => site,
   site => {
     const {favicon, parsedUrl, label} = selectSiteProperties(site);
-    const backgroundRGB = getBackgroundRGB(site);
-    const backgroundColor = site.background_color || toRGBString(...backgroundRGB, favicon ? BACKGROUND_FADE : 1);
+    const backgroundRGB = site.background_color || [...getBackgroundRGB(site), favicon ? BACKGROUND_FADE : 1];
+    const backgroundColor = toRGBString(backgroundRGB);
     const fontColor = getBlackOrWhite(...backgroundRGB);
     const firstLetter = prettyUrl(parsedUrl.hostname)[0];
     return {
```

Comment:
I think this file could use some further consolidation, but doesn't have to be in this patch. Like maybe the fallback color stuff could just be in the same place as the other background color stuff.

Anyways, I was kinda confused, but it checks out 👍